### ISSUE TITLE
Flexed position of finger (not a fixed contracture)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10688]
**Preferred term label:**
Flexed position of finger (not a fixed contracture)

**Synonyms**

**Definition (free text, please give PubMed ID)**
flexed position but it can be extended, though that may be painful

**Parent term (use hpo.jax.org/app)**
Sibling term is contracture, but this is maintained flexed not due to a contracture but rather because of some other factor such as pain on extension because of a infection at the tendon of a finger

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pyogenic flexor tenosynovitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Perhaps "Flexed posturing of finger" would be a better term label?
Definition: A sign that can be observed in diseases such as Pyogenic or suppurative flexor tenosynovitis in which the affected finger is held in a flexed position. Often, there is pain on any attempt at passive finger extension. This finding is to be distinguished from a contracture of a finger joint in which the flexed positions is fixed.
PMID: [35015439](https://pubmed.ncbi.nlm.nih.gov/35015439)


See:
https://www.ncbi.nlm.nih.gov/books/NBK576414/
and https://www.orthobullets.com/hand/6105/pyogenic-flexor-tenosynovitis
Kanavel signs (4 total) 
 
flexed posturing of the involved digit
tenderness to palpation over the tendon sheath
marked pain with passive extension of the digit
fusiform swelling of the digit

**Comment by MickeySegal** [comment]
This is good.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001241
- label: Flexed posturing of finger
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: PMID:35015439
- def: A sign that can be observed in diseases such as Pyogenic or suppurative flexor tenosynovitis in which the affected finger is held in a flexed position. Often, there is pain on any attempt at passive finger extension. This finding is to be distinguished from a contracture of a finger joint in which the flexed positions is fixed.
- comment: This finding is one of the four Kanavel signs for pyogenic flexor tenosynovitis: (i) flexed posturing of the involved digit; (ii) tenderness to palpation over the tendon sheath; (iii) marked pain with passive extension of the digit; and (iv) fusiform swelling of the digit.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10688
- synonyms:
